ddtrace/contrib/aiohttp/__init__.py

```diff
@@ -1,22 +1,42 @@
 """
-The ``aiohttp`` integration traces all requests received by defined routes
-and handlers. External modules for database calls and templates rendering
-are not automatically instrumented, so you must use the ``patch()`` function::
+The ``aiohttp`` integration traces all requests defined in the application handlers.
+Auto instrumentation is available through a middleware and a ``on_prepare`` signal
```

Comment:
Do we have to talk about `on_prepare`? Looks like an internal thing. If users don't have to care about it, let's not talk about it.